### PR TITLE
Change all mentions of "semi-colon" to "colon"

### PR DESCRIPTION
**Pull request by noplanman**

### CODE REVIEW

File: tests/HOTPTest.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ public function testIssuerHasSemicolon()
 
     /**
      * @expectedException \InvalidArgumentException
-     * @expectedExceptionMessage Issuer must not contain a semi-colon.
+     * @expectedExceptionMessage Issuer must not contain a colon.
      */
     public function testIssuerHasSemicolon2()
```
Comment (Spomky) [inline tests/HOTPTest.php]: This method name (and the two others below this one) may be renamed.

**Comment by Spomky** [comment]
Thanks.
I have just added one comment (method names in test cases could be renamed).
Otherwise this PR looks good.

**Comment by noplanman** [comment]
@Spomky Totally overlooked those method names, sorry.

Have added a commit.

**Comment by Spomky** [comment]
Thanks!

**Pull request merged**